flowzone.yml

```diff
@@ -1289,13 +1289,13 @@ jobs:
       (
         github.event.action != 'closed' ||
         github.event.pull_request.merged == true
-      ) &&
-      inputs.release_notes == true
+      )
```

Comment:
I would prefer we keep this job as opt-in, especially with how many changes it's been getting recently.